app/src/main/res/layout/list_stream_item.xml

```diff
@@ -23,6 +23,37 @@
         android:src="@drawable/dummy_thumbnail"
         tools:ignore="RtlHardcoded"/>
 
+    <FrameLayout
```

Comment:
Just a question: why ia a `FrameLayout` being used here, thus encapsulating the below `TextView`?

Reply:
I think I was testing something back then. Removed it.